app/tests/conftest.py

```diff
@@ -760,6 +760,10 @@ def record_files():
         FileReference="ABCDE",
         FilePath="data/content/test_folder/open_file.docx",
         CiteableReference="first_body/ABCDE",
+        ffid_metadata__Extension="docx",
```

Comment:
Not needed by tests as we are using Factories instead of this fixture but added for consistency